src/core/lib/gpr/time.cc

```diff
@@ -236,6 +236,30 @@ double gpr_timespec_to_micros(gpr_timespec t) {
   return static_cast<double>(t.tv_sec) * GPR_US_PER_SEC + t.tv_nsec * 1e-3;
 }
 
+// TABLE[src][dst] = Tdst - Tsrc
+static gpr_timespec gpr_clock_offset_table[3][3];
+
+void grpc_build_clock_offset_table() {
```

Comment:
AFAIR values of some clock types are calculated from other clock types. Is it guaranteed that the offset table will still work fine even after an application has been running for long time? (e.g. can realtime clock run at a slightly different speed than precise?).

Reply:
You're right. It's possible to have those clocks having slightly different speed. Rather, I'm going to think of other approach fixing this problem. Thanks!